src/applicants/applicant.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {Entity, Column, PrimaryGeneratedColumn} from "typeorm"
+
+@Entity()
+export class Applicant {
+    @PrimaryGeneratedColumn('uuid')
+    id: string
+
+    @Column({
+        unique: true
+    })
+    email: string;
+
+
+    @Column({
```

Comment:
i mentioned that the api only receives email and category... why are u adding a compulsory field for gender here? @SunmadeBabafemi

Reply:
i did not add gender this was the first  api.. the correct api is the waitlist.. let me do some cleaning of old codes.. sorry